lib/rgeo/coord_sys.rb

```diff
@@ -24,6 +25,9 @@ module RGeo
   # systems, and other related concepts, as well as a parser for the WKT
   # format for specifying coordinate systems.
   module CoordSys
+    CONFIG = OpenStruct.new
```

Comment:
I think `Struct` is better perf-wise:

```ruby
# frozen_string_literal: true

require "ostruct"
require "benchmark"
require "benchmark/ips"

def h1(str)
  puts
  puts str.upcase
  puts ?= * str.size
  puts
end

STRUCT = Struct.new(:foo, keyword_init: true)

h1 "creation"

Benchmark.ips do |x|
  x.report("Struct") { STRUCT.new(foo: "bar") }
  x.report("OpenStruct") { OpenStruct.new(foo: "bar") }
  x.compare!
end

h1 "record access"

struct = STRUCT.new(foo: "bar")
ostruct = OpenStruct.new(foo: "bar")

Benchmark.ips do |x|
  x.report("Struct") { struct.foo }
  x.report("OpenStruct") { ostruct.foo }
  x.compare!
end
```

```
CREATION
========

Warming up --------------------------------------
              Struct   533.208k i/100ms
          OpenStruct    27.978k i/100ms
Calculating -------------------------------------
              Struct      5.369M (± 2.1%) i/s -     27.194M in   5.067529s
          OpenStruct    281.461k (± 2.2%) i/s -      1.427M in   5.072052s

Comparison:
              Struct:  5368589.4 i/s
          OpenStruct:   281460.9 i/s - 19.07x  (± 0.00) slower


RECORD ACCESS
=============

Warming up --------------------------------------
              Struct     2.084M i/100ms
          OpenStruct   970.030k i/100ms
Calculating -------------------------------------
              Struct     21.106M (± 0.5%) i/s -    106.283M in   5.035739s
          OpenStruct      9.748M (± 0.6%) i/s -     49.472M in   5.075373s

Comparison:
              Struct: 21106238.1 i/s
          OpenStruct:  9747667.8 i/s - 2.17x  (± 0.00) slower
```

Although, accessing the value is not that slower in the end. However, not requiring ostruct is also a benefit, don't you think?

```suggestion
    CONFIG = Struct.new(:default_coord_sys_class).new(CS::CoordinateSystem)
```

IDK if this would be enough ?